### DISCUSSION TITLE
Mirror or steal from multiple containers

### DISCUSSION THREAD

**Discussion by tamasfe** [metalbear-co/mirrord#541] [Ideas]
Would it be feasible to mux network from multiple containers at the same time? This way all replicas of a deployment could be impersonated at the same time, it would also allow for services to be impersonated by impersonating each associated pod.

Some features (such as fs emulation) is not possible as it does not make much sense, but as a user, I would expect this behaviour when specifying deployments as a target.

**Comment by eyalb181** [comment]
Hey @tamasfe, thanks for bringing this up. It's definitely on the roadmap, quite a big project though. We'll be adding this as a temporary solution (kind of) in the meanwhile:
https://github.com/metalbear-co/mirrord/issues/527
